tests/test_interactions.py

```diff
@@ -263,7 +263,7 @@ def test_scores(self):
 
             # calculate manually
             scores_f_manual = self._exp_score(**kwargs).view(-1)
-            assert torch.allclose(scores_f_manual, scores_f)
+            assert torch.allclose(scores_f_manual, scores_f), f'Diff: {scores_f_manual - scores_f}'
```

Comment:
Locally, for me, using `py.test` shows the arguments that lead to failure one line above the assertion error,